app/views/collections/edit.html.erb

```diff
@@ -16,8 +16,14 @@
   </div>
 <% end %>
 
-<h2>Manage Items in this Collection</h2>
-<%= render 'search_collection_dashboard_form'%>
+<br/>
+<br/>
+<div class="row">
+  <h2 class="col-xs-12 col-sm-6"><%= t('Manage Items in this Collection') %></h2>
```

Comment:
Yep.  I don't have a populated translation, but it gives me back the parameter string at least.